app/assets/javascripts/select-kit/addon/components/icon-picker.js

```diff
@@ -98,28 +102,27 @@ export default MultiSelectComponent.extend({
       name: iconName,
       icon: strippedIconName,
     };
-  },
+  }
 
   willDestroyElement() {
     $("#svg-sprites .ajax-icon-holder").remove();
-    this._super(...arguments);
+    super.willDestroyElement(...arguments);
 
     this._cachedIconsList = null;
     this._resultCount = 0;
 
     if (isDevelopment()) {
       enableMissingIconWarning();
     }
-  },
+  }
 
-  actions: {
-    onChange(value, item) {
-      if (this.selectKit.options.maximum === 1) {
-        value = value.length ? value[0] : null;
-        item = item.length ? item[0] : null;
-      }
+  @action
+  _onChange(value, item) {
```

Comment:
Renamed to avoid clash with the @onChange argument. This works because select-kit will call the underscored version of actions first.